docs/src/environments/_index.md

```diff
@@ -138,3 +138,22 @@ Staging
   Developer2
     Ticket-395
 ```
+
+## Environment status
+
+Your environments can have one of two statuses:
+
+* [Active](../other/glossary.md#active-environment):
+  A deployed environment with services and data.
+* [Inactive](../other/glossary.md#inactive-environment):
+  An environment that isn't deployed and has no services or data, only code.
+
+You can see an environment's status in multiple ways:
+
+* In the console:
+  * Inactive environments are lighter in the environment list and don't appear in the environment selector.
```

Comment:
:nail_care:
We could add the link to [the environment configuration](https://pr-2328-nkae3aq-652soceglkw4u.eu-3.platformsh.site/administration/web/configure-environment.html).

Reply:
I added it to the top-level list item, if you think that works.

Reply:
Yes, that works perfectly, thanks 🙏🏼